x-pack/plugin/sql/src/test/java/org/elasticsearch/xpack/sql/expression/function/scalar/datetime/DateTimeParseProcessorTests.java

```diff
@@ -192,14 +189,14 @@ public void testParsing() {
         assertEquals(
                 time(10, 20, 30, 123000000, zoneId),
                 new TimeParse(Source.EMPTY, l("10:20:30.123"), l("HH:mm:ss.SSS"), zoneId).makePipe()
-                        .asProcessor()
-                        .process(null)
+                    .asProcessor()
+                    .process(null)
```

Comment:
All these four lines do not align with the rest of the calls in the same method. Compare these with the ones above and, please, change them to have a consistent look and feel.